scenario/scenario_shacl.ttl

```diff
@@ -12,95 +12,54 @@ scenario:ScenarioShape a sh:NodeShape ;
     sh:property [ sh:maxCount 1 ;
             sh:minCount 1 ;
             sh:node general:GeneralShape ;
-            sh:name "general object" ;
+            sh:name "General Object" ;
             sh:description "general object with properties for descriptions, data, links, bundle" ;
-            sh:order 5 ;
+            sh:order 1 ;
             sh:path scenario:general ],
         [ sh:maxCount 1 ;
             sh:minCount 1 ;
-            sh:node scenario:EnvironmentalShape ;
-            sh:name "environmental object" ;
-            sh:description "environmental object with property for sunAzimuth" ;
+            sh:node scenario:FormatShape ;
+            sh:name "Format Object" ;
```

Comment:
in den anderen domains, wie z. B. HdMap haben die Kollegen Kleinbuchstaben verwendet, z. B. 

![image](https://github.com/user-attachments/assets/21d1cb85-35f1-4044-8687-a89cf166baeb)

Der Einheitlichkeit halber schlage ich vor, das hier auch zu machen.

Reply:
Hab ich umgesetzt. Denke zwar (weil `sh:name` menschenlesbar sein soll), dass upper case generell sinnvoller wäre aber hier in dem Fall (weil es bei allen anderen lower case ist) passt das schon :D